.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Main tag created workflow
+
+on:
+  push:
+    tags-ignore:
+    - 'newtonsoft-*'
```

Comment:
This makes sure that it doesn't run for when we want to release a new version of the Newtonsoft package

Reply:
Is that looking for a tag containing the term "newtonsoft"

Reply:
Apologies, that should have read "for when we want to push a release of the Carter package".

And you're correct, this is ignoring tags that start with `newtonsoft`

Reply:
Thing is tags will probably just be `v7.0.0` for example

Reply:
I used different tags as I though you wanted to be able to have different versions of the `Carter` and the `Carter.ResponseNegotiators.Newtonsoft` packages while the source stays in the same repo.

"Regular" tags, like `v7.0.0`, would release `Carter`, while `newtonsoft-v7.0.0`, would release the `Carter.ResponseNegotiators.Newtonsoft` package.

Did I make a wrong assumption thinking you might want to keep their versioning separate?
My thinking was that if we keep them in sync, new versions of `Carter.ResponseNegotiators.Newtonsoft` could be released without any changes in them.